### PR TITLE
Align casing of file and folder names resolves #57

### LINKED ISSUE #57: The consistent naming convention in the repo is camelCase for directories and files
The consistent naming convention in the repo is camelCase for directories and files so I would rename:

`config-manager` -> `configManager`
`default-config` -> `defaultConfig`

_Originally posted by @brafdlog in https://github.com/brafdlog/budget-tracking/pull/54#issuecomment-643321496_

I think the Vue part is not full compatible with this rule. But also need to think maybe in Vue the convention is different- https://vuejs.org/v2/style-guide/#Single-file-component-filename-casing-strongly-recommended

### PR DESCRIPTION
**Pull request by brafdlog**

### CODE REVIEW

**Comment by brafdlog** [comment]
For #57

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -70,6 +70,7 @@ jobs:
         GOOGLE_CLIENT_ID: ${{ secrets.GOOGLE_CLIENT_ID }}
         GOOGLE_CLIENT_SECRET: ${{ secrets.GOOGLE_CLIENT_SECRET }}
         GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        NODE_OPTIONS: --max-old-space-size=4096
```
Comment (baruchiro) [inline .github/workflows/release.yml:73]: Did you have any problems?

**Review by baruchiro — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (brafdlog) [reply .github/workflows/release.yml:73]: Yes. The release crashed on out of memory

Comment (baruchiro) [reply .github/workflows/release.yml:73]: Interesting, what has changed?

Comment (brafdlog) [reply .github/workflows/release.yml:73]: Don't know... But the mac release is relatively new and oom issues can happen inconsistently if it gets close to the limit

Comment (baruchiro) [reply .github/workflows/release.yml:73]: Oh, Mac release, OK

**Pull request merged**